### PR TITLE
Add query & write series API definitions.

### PR DESCRIPTION
**Pull request by tiraffe**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+          "description": "Timestamp of a data point (ISO8601 format)."
+        },
+        "value": {
+          "type": "number",
+          "format": "float",
+          "description": "The measurement of that point, should be float."
+        }
+      }
+    },
+    "Timeseries": {
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: make this called "Points"

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+            "$ref": "#/parameters/timeseriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Request body for writing timeseries.",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/Timeseries"
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: Timeseries -> Points

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+            "$ref": "#/parameters/timeseriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Request body for querying timeseries.",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/QueryRequest"
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: change to TimeSeriesQueryRequest

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation.",
+            "schema": {
+              "$ref": "#/definitions/QueryResponse"
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: TimeSeriesQueryResponse

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/APIError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Query Timeseries example": {
+            "$ref": "./examples/QuerySeries.json"
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: TimeSeriesQuery.json

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+        },
+        "end": {
+          "format": "date-time",
+          "type": "string",
+          "description": "Timestamp of end time (ISO8601 format)."
+        },
+        "fields": {
+          "type": "array",
+          "description": "Name of timeseries field.",
+          "items": {
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json:119]: define a Enum type please. Refer to https://github.com/moreOver0/azure-rest-api-specs/blob/43d7d1914f04983be88346e7bd33723d9ef4393c/specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/AnomalyDetector.json#L293

Comment (moreOver0) [reply specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json:119]: no "none"

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+        "series": {
+          "type": "array",
+          "description": "Time series data points.",
+          "items": {
+            "$ref": "#/definitions/Point"
+          }
+        }
+      }
+    },
+    "QueryRequest": {
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: change name

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 8]
```diff
@@ -0,0 +1,196 @@
...
+              "upperMargin",
+              "lowerMargin",
+              "period"
+            ],
+            "type": "string"
+          }
+        }
+      }
+    },
+    "QueryResponse": {
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: change name

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+          "description": "Time series data points.",
+          "items": {
+            "$ref": "#/definitions/Point"
+          }
+        }
+      }
+    },
+    "QueryRequest": {
+      "type": "object",
+      "properties": {
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: "required" list

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+              "period"
+            ],
+            "type": "string"
+          }
+        }
+      }
+    },
+    "QueryResponse": {
+      "type": "object",
+      "properties": {
```
Comment (moreOver0) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json:151]: "required" list

**Review by moreOver0 — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 11]
```diff
@@ -0,0 +1,216 @@
+{
+  "paths": {
+    "/timeseries/{timeseriesId}/write": {
+      "put": {
+        "description": "Insert or replace timeseries data for specified timeseriesId. Note that this interface only stores data and do not detect these data.",
+        "operationId": "Timeseries_Write",
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: TimeSeries_Write_Post

Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: TimeSeries_Write

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 12]
```diff
@@ -0,0 +1,216 @@
...
+          "Write Timeseries example": {
+            "$ref": "./examples/TimeSeriesWrite.json"
+          }
+        }
+      }
+    },
+    "/timeseries/{timeseriesId}/query": {
+      "post": {
+        "description": "Query timeseries with required field in each timestamp.",
+        "operationId": "Timeseries_Query",
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: TimeSeries_Query

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 13]
```diff
@@ -0,0 +1,216 @@
...
+        "x-ms-examples": {
+          "Query Timeseries example": {
+            "$ref": "./examples/TimeSeriesQuery.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Point": {
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: Can use the original Point in AnomalyDetection.json

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 14]
```diff
@@ -0,0 +1,216 @@
...
+            "$ref": "#/parameters/timeseriesId"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "description": "Request body for writing timeseries.",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/Points"
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: {
           "type": "array",
          "description": "Time series data points.",
          "items": {
            "$ref": "#/definitions/Point"
          }

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 15]
```diff
@@ -0,0 +1,216 @@
...
+          "description": "Timestamp of a data point (ISO8601 format)."
+        },
+        "value": {
+          "type": "number",
+          "format": "float",
+          "description": "The measurement of that point, should be float."
+        }
+      }
+    },
+    "Points": {
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: remove points

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json [hunk 16]
```diff
@@ -0,0 +1,216 @@
...
+            "items": {
+              "type": "object"
+            }
+          }
+        }
+      }
+    }
+  },
+  "parameters": {
+    "timeseriesId": {
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/QueryWriteSeries.json]: remove this we will have a common one

File: specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/examples/TimeSeriesWrite.json [hunk 17]
```diff
@@ -0,0 +1,205 @@
+{
+  "parameters": {
+    "Endpoint": "{Endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "timeseriesId": "abc",
+    "body": {
+      "series": [
```
Comment (conhua) [inline specification/cognitiveservices/data-plane/AnomalyDetector/preview/v1.0/examples/TimeSeriesWrite.json]: remove series, just a list of points

**Pull request merged**